### ISSUE TITLE
Accessibility: Presentational Attributes Used

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#214]
I'm coming across an error on all pages for "Presentational Attributes Used" for the following code:
`<body class="page-template-default page page-id-1346 page-child parent-pageid-1329 _masterslider _msp_version_3.1.3 sidebar-left" style="position: relative; min-height: 100%; top: 0px;">...</body>`

Here is the description of the error:
Presentational attributes used
1.3.1 Info and Relationships

Presentational attributes such as 'border', 'align', or 'bgcolor' are used. CSS should be used for styling instead. It's important to review these issues and determine if they can be fixed by moving the presentational attributes to your CSS. It's not always practical to make these changes, but should be made when possible.

Thanks!!

**Comment by joelworsham** [comment]
This warning will need to be ignored. It caught this potential issue in a broad-sweeping way, but this use-case is not "bad". The attributes in question are automatically added by the Google Translate plugin and cannot be prevented.

I would add this one to the whitelist, if you can.
